Dockerfile

```diff
@@ -11,6 +11,10 @@ WORKDIR /src
 # don't care for our tests.
 ENV CASS_DRIVER_NO_EXTENSIONS theytaketoolongtobuild
 
+RUN apt-get update -y
```

Comment:
FYI -I needed to add these for the building the kafka connector -there were header files missing and connector could not be built without the librdkafka-dev